src/components/add-journey/JourneySearchForm.tsx

```diff
@@ -7,6 +7,8 @@ import { StationSearchField } from '@/components/add-journey/StationSearchField'
 import { useJourneySearchStore } from '@/hooks/useJourneySearchStore';
 import { classNames } from '@/utils/styling';
 import { PlusIcon } from '@heroicons/react/outline';
+import { FlightTimeToggle } from './FlightTimeToggle';
```

Comment:
Please use an absolute import path since that's the code standard in this repository.

Like this:
`'@/components/add-journey/...';`